tests/unit/test_routes.py

```diff
@@ -93,11 +93,24 @@ def test_single_resource_out_of_bounds(module_client, module_db):
 
     assert (response.status_code == 404)
 
+def test_get_docs(module_client):
+    response = module_client.get("/")
+    assert (response.status_code == 200)
+
+def test_open_api_yaml(module_client):
+    response = module_client.get("/openapi.yaml")
+    assert (response.status_code == 200)
+
 
 def test_get_favicon(module_client):
     response = module_client.get("favicon.ico")
     assert (response.status_code == 200)
 
+def dir_last_updated(folder):
+    response = module_client.get("favicon.ico")
+    assert (response.status_code == 200)
```

Comment:
This isn't really testing the `dir_last_updated` function. `dir_last_updated` is used in the `/` route, as shows here:
https://github.com/OperationCode/resources_api/blob/4433a9471c25db88f737b29dd38ee57e73457c1b/app/views/routes.py#L16

It's a variable that's available to index.html, as shown by the flask documentation here https://flask.palletsprojects.com/en/1.1.x/api/#flask.render_template (see `context`).

Anyway, the real way to get a behavior-driven test around this function would be to call the function and save the timestamp it returns, then change the file and call the function again and note that the timestamp has changed. However, that doesn't sound too practical for a unit test 🤔

Reply:
hey man how do you debug pytest in docker? For example I am trying to understand the module_client fixture and what content it holds in a test. For example module_client.get("")..and what the response object holds?

Reply:
Personally, I debug with print statements, but I should probably get a real debugger one day 😅

Reply:
same, but for example if i want to print a variable in a fixture in conftest? Instead of using make, i run 

```
docker-compose run resources_api py.test /tests/conftest.py
```
so i imagine you might not test with docker since you created all this and runs locally on your box without docker. though in the situation of testing within a container, i was curious how you printed and debugged.

Reply:
When I run `make test` I see the output from print statements even though it uses docker. Sometimes when testing, print statements don't execute because they only print on failures, so I'll do something like `raise exception` right after my print statement so I can see the print. It's not an ideal flow, I think it would be better to get pydebugger or similar working with docker, but I haven't really had the time/motivation for such a task.

If you get something working, please report back (or better yet add something to the README about how to debug tests 😄)